src/benchmarklib/benchmark_utils.cpp

```diff
@@ -68,4 +77,140 @@ BenchmarkConfig::BenchmarkConfig(const BenchmarkMode benchmark_mode, const bool
       enable_visualization(enable_visualization),
       out(out) {}
 
+BenchmarkConfig BenchmarkConfig::get_default_config() { return BenchmarkConfig(); }
+
+bool CLIConfigParser::cli_has_json_config(const int argc, char** argv) {
+  const auto has_json = argc > 1 && boost::algorithm::ends_with(argv[1], ".json");
+  if (has_json && argc > 2) {
+      std::cout << "Passed multiple args with a json config. All CLI args will be ignored...";
+  }
+
+  return has_json;
+}
+
+nlohmann::json CLIConfigParser::config_file_to_json(const std::string& json_file_str) {
+  Assert(filesystem::is_regular_file(json_file_str), "No such file " + json_file_str);
+
+  nlohmann::json json_config;
+  std::ifstream json_file{json_file_str};
+  json_file >> json_config;
+
+  return json_config;
+}
+
+BenchmarkConfig CLIConfigParser::parse_default_json_config(const nlohmann::json& json_config) {
+  const auto default_config = BenchmarkConfig::get_default_config();
+
+  // Should the benchmark be run in verbose mode
+  const auto verbose = json_config.value("verbose", default_config.verbose);
+  auto& out = get_out_stream(verbose);
+
+  // In non-verbose mode, disable performance warnings
```

Comment:
The warning that Hyrise was built in debug mode still appears, right?

Reply:
yes. but thanks for the pointer. there was an error here that i had to fix